src/app/pages/AccountDetailsPage/index.tsx

```diff
@@ -71,26 +64,22 @@ export const AccountDetailsPage: FC = () => {
           tokenPriceInfo={tokenPriceInfo}
         />
       </SubPageCard>
-      {showDetails && (
-        <RouterTabs
-          tabs={[
-            { label: t('common.transactions'), to: txLink, visible: showTxs },
```

Comment:
Simplify RouterTabs too?

Reply:
I think being able to hide tabs is a generally useful feature (for the tab menu), so I would't remove it unless it becomes uncomfortable to maintain for some reason.

(Also, please note that this feature is better than simply _not_ listing an item as part of the menu, because as implemented now, an "invisible" tab can still be validly selected and displayed when accessing via a link, just not offered up for selection by default.)